tutorials/cloudsdk-ci-cd/cloudsdk-ci-cd.md

```diff
@@ -0,0 +1,252 @@
+---
+author_name: Benjamin Heilbrunn
+title: Cloud SDK Continuous Integration and Delivery
+description: Learn how to use the SAP Cloud SDK Continuous Integration and Continuous Delivery toolkit to build, test, and deploy your project.
+time: 30
+tags: [tutorial>intermediate, products>sap-s-4hana-cloud-sdk]
+primary_tag: products>sap-s-4hana-cloud-sdk
+---
+
+## Prerequisites
+For getting started with Continuous Integration and Delivery (`Cx`) in your project, you need to assure the following prerequisites.
+**Linux host with Docker**: For instantiating the SAP Cloud SDK Cx-Server, you need to provide a suitable host or virtual machine with a Linux operating system and Docker installed. Please also ensure that the user with whom you start the Cx-Server belongs to the docker group.
+**Project sources in GitHub or Git**: Your project source files need to be available on a git or `Github server`, which is accessible from the Cx-Server host. Creating your project is explained in [Step 2](https://blogs.sap.com/2017/05/21/step-2-with-sap-s4hana-cloud-sdk-helloworld-on-scp-classic/) and [Step 3](https://blogs.sap.com/2017/05/19/step-3-with-sap-s4hana-cloud-sdk-helloworld-on-scp-cloudfoundry/).
+
+>The SAP Cloud SDK Pipeline uses Docker images for each individual build step. Accordingly, you do not need to take care of installing any further dependencies on your host machine. All tools required for building, testing, quality checking, and deploying your applications, are dynamically retrieved in the form of Docker images.
+
+## Details
+### You will learn
+  - How to develop and release your application in short cycles.
+  - How to install and run cx-server.
+  - How to setup a Jenkins job.
+
+---
+
+[ACCORDION-BEGIN [Step 1: ](Installing and Managing the Cx-Server)]
```

Comment:
All steps are simple verbs and sentence case, so make this Install and manage ... 

Do this for other steps.

Step 1 is quite long -- can we divide into several steps?